src/sagemaker/jumpstart/estimator.py

```diff
@@ -39,128 +33,78 @@ class JumpStartEstimator(Estimator):
 
-        default_metric_definitions = metric_definitions.retrieve_default(
-            region=region, model_id=model_id, model_version=model_version
+        estimator_fit_kwargs = get_fit_kwargs(
+            model_id=self.model_id,
+            model_version=self.model_version,
+            instance_type=self.instance_type,
+            instance_count=self.instance_count,
+            region=self.region,
+            kwargs=kwargs,
         )
 
-        curr_metric_definitions = self.kwargs_for_base_estimator_class.get("metric_definitions", [])
-        new_metric_definitions = deepcopy(curr_metric_definitions)
-
-        for metric_definition in default_metric_definitions:
-            if metric_definition["Name"] not in [
-                definition["Name"] for definition in new_metric_definitions
-            ]:
-                new_metric_definitions.append(metric_definition)
-
-        if new_metric_definitions == []:
-            new_metric_definitions = None
+        return super(JumpStartEstimator, self).fit(*largs, **estimator_fit_kwargs.to_kwargs_dict())
 
-        self.kwargs_for_base_estimator_class["metric_definitions"] = new_metric_definitions
-
-        # estimator_kwargs_to_add = _retrieve_kwargs(model_id=model_id, model_version=model_version, region=region)
-        estimator_kwargs_to_add = {}
-
-        new_kwargs_for_base_estimator_class = deepcopy(self.kwargs_for_base_estimator_class)
-        for key, value in estimator_kwargs_to_add:
-            new_kwargs_for_base_estimator_class = update_dict_if_key_not_present(
-                new_kwargs_for_base_estimator_class,
-                key,
-                value,
-            )
-
-        self.kwargs_for_base_estimator_class = new_kwargs_for_base_estimator_class
-
-        self.kwargs_for_base_estimator_class["model_id"] = model_id
-        self.kwargs_for_base_estimator_class["model_version"] = model_version
-
-        # self.kwargs_for_base_estimator_class = update_dict_if_key_not_present(
-        #     self.kwargs_for_base_estimator_class,
-        #     "predictor_cls",
-        #     JumpStartPredictor,
-        # )
-
-        self.kwargs_for_base_estimator_class = update_dict_if_key_not_present(
-            self.kwargs_for_base_estimator_class, "instance_count", 1
-        )
-        self.kwargs_for_base_estimator_class = update_dict_if_key_not_present(
-            self.kwargs_for_base_estimator_class,
-            "instance_type",
-            instance_types.retrieve_default(
-                region=region, model_id=model_id, model_version=model_version
-            ),
+    def deploy(
+        self,
+        image_uri: Optional[str] = None,
+        source_dir: Optional[str] = None,
+        entry_point: Optional[str] = None,
+        env: Optional[Dict[str, str]] = None,
+        predictor_cls: Optional[Predictor] = None,
+        initial_instance_count: Optional[int] = None,
+        instance_type: Optional[str] = None,
```

Comment:
please also expose `wait`, `kms_key`, `tags`, `model_download_timeout`, `container_startup_health_check_timeout` and `volume_size`  from the [deploy](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/estimator.py#L112) method.

Also `image_uri`, `source_dir`, `entry_point`, `env` and `predictor_cls` are inconsistent, please remove them. `deploy` seems to pass `serializer` and `deserializer`, that is what we should accept.